workflow/layout/hofx_only.yaml

```diff
@@ -0,0 +1,48 @@
+suite: !Cycle
+      <<: *suite_defaults
+
+      obs_prep: !Task
+        <<: *shared_task_template
+        Trigger: !Depend  ( ~ suite.has_cycle('-24:00:00') | hofx_run.at('-24:00:00'))
+        resources: !calc partition.resources.run_obs_prep
+        config: [ base ]
+        J_JOB: JJOB_OBS_PREP
+
+      hofx_prep: !Task
+        <<: *shared_task_template
+        Trigger: !Depend  obs_prep
+        resources: !calc partition.resources.run_fcst_prep
+        config: [ base ]
+        J_JOB: JJOB_HOFX_PREP
+
+      da_prep_soca: !Task
```

Comment:
That one step is over-kill, I guess this is fine for this pr, but we need to document the problem in the issues. This will generate the grid, which is needed, but also compute the bump coef, not needed and resource intensive ~5-10 mn on 400 cores for the 1/4 degree case.

Reply:
Agreed. Making the change.